exercises/crypto-square/canonical-data.json

```diff
@@ -1,60 +1,60 @@
 {
   "exercise": "crypto-square",
-  "version": "3.2.0",
+  "version": "3.2.1",
   "cases": [
     {
       "description": "empty plaintext results in an empty ciphertext",
```

Comment:
I think this is good. I think "plaintext" and "ciphertext" are better in descriptions than "plain text" and "cipher text"